src/pages/Launchpad/Progress.tsx

```diff
@@ -99,13 +103,14 @@ function Step({
             isDone ? "bg-orange" : "bg-gray-l3 dark:bg-bluegray"
           } rounded-full transform -translate-x-1/2`}
         />
-        <span
+        <Link
```

Comment:
according to ticket, link only works if `isDone` - though there's a check that redirects back to latest progress, its purpose is to guard users from inputing `step` directly from url